bookshelf/test/lib/GetConfigTrait.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Samples\Bookshelf;
+
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Class GetConfigTrait
+ * @package Google\Cloud\Samples\Bookshelf
+ *
+ * Use this trait to load the project configuration
+ */
+trait GetConfigTrait
+{
+    protected function getConfig()
+    {
+        $config = [];
+
+        // if the settings.yml file exists, copy it over
+        if (file_exists(__DIR__ . '/../../config/settings.yml')) {
```

Comment:
My reasoning here was for config items that we don't need to change w/ EnvVars (i.e. `bookshelf_backend`. Also, if a user sets their local config, they can run tests against it.

I think it's better to keep it, but I'm mostly ambivalent. Is your argument here for keeping it simple?